Test/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 add_subdirectory(list)
 add_subdirectory(utils)
+add_subdirectory(vector)
+add_subdirectory(btree)
```

Comment:
What's the difference

Reply:
BST is a special kind of binary tree where the condition follows that `left < root < right`.

A binary tree is simply a tree with at most two children.

I haven't decided the APIs to manipulate a binary tree.

As for BST, it can be extended to include non `int` data as well. Though leave that to me.